app/src/main/java/fr/neamar/kiss/SettingsActivity.java

```diff
@@ -26,7 +26,7 @@
         SharedPreferences.OnSharedPreferenceChangeListener {
 
     // Those settings require the app to restart
-    final static private String requireRestartSettings = "theme enable-spellcheck force-portrait";
+    final static private String requireRestartSettings = "theme enable-spellcheck force-portrait enable-favorites-bar";
```

Comment:
Is there a technical reason why `enable-favorites-bar` *must* require a restart? Showing/hiding the favorites bar can also be done in `onResume` and calling each time the launcher is resumed is probably no big burden either (and, if I interpret the code correctly) makes the launcher respond better to app icon changes from themes, upgrades, …